demos/sslecho/.project

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file

Reply:
Have removed both Eclipse project files.